src/Connector.SqlServer/Features/ITimeStampingFeature.cs

```diff
@@ -0,0 +1,12 @@
+﻿using CluedIn.Connector.SqlServer.Connector;
+using CluedIn.Core.Connectors;
+using CluedIn.Core.Streams.Models;
+using System.Threading.Tasks;
+
+namespace CluedIn.Connector.SqlServer.Features
+{
+    public interface ITimeStampingFeature
```

Comment:
Was renamed from 'IUpgradeExistingSchemaFeature'

Reply:
We have class named `DefaultUpgradeExistingSchemaFeature` with method `VerifyExistingContainer` which actually check table columns and add TimeStamp column if it's missing.

These names does not correspond to what it actually do. This only brings some chaos into the package.

Reply:
@mccartney1977 also, what existing implementations are we taking into considerations?